bot_core/commands.py

```diff
@@ -10,6 +11,13 @@ class MovieCommands:
     def __init__(self, movie_night_manager, movie_night_service):
         self.movie_night_manager = movie_night_manager
         self.movie_night_service = movie_night_service
+    
+    async def parse_movie_urls(self, movie_urls):
```

Comment:
The parse_movie_urls function isn't awaited in your code and isn't IO-bound. Remove async for it to work correctly and improve efficiency.